e2e-tests/playwright/e2e/configuration-test/config-map.spec.ts

```diff
@@ -4,19 +4,20 @@ import { LOGGER } from "../../utils/logger";
 import { Common } from "../../utils/common";
 import { UIhelper } from "../../utils/ui-helper";
 
-test.describe.skip("Change app-config at e2e test runtime", () => {
+test.describe("Change app-config at e2e test runtime", () => {
   test("Verify title change after ConfigMap modification", async ({ page }) => {
-    test.setTimeout(300000); // Increasing to 5 minutes
+    test.setTimeout(900000); // Increased to 15 minutes
```

Comment:
That is a pretty long timeout, and the test passed in 2 minutes. Do we need it that high? Or is it a leftover from debugging?